js/plugins/vertexai/src/gemini.ts

```diff
@@ -70,26 +70,55 @@ const SafetySettingsSchema = z.object({
 });
 
 const VertexRetrievalSchema = z.object({
-  datastore: z.object({
-    projectId: z.string().optional(),
-    location: z.string().optional(),
-    dataStoreId: z.string(),
-  }),
-  disableAttribution: z.boolean().optional(),
+  datastore: z
+    .object({
+      projectId: z
+        .string()
+        .describe('Project ID of the data store.')
+        .optional(),
+      location: z.string().describe('Location of the data store.').optional(),
+      dataStoreId: z
+        .string()
+        .describe(
+          'Data store resource name. Must provide either the full ' +
+            "data store resource name here as: 'projects/{project}/" +
+            'locations/{location}/collections/default_collection/' +
+            "dataStores/{data_store}' or provide {data_store} ID here " +
+            'and provide project ID, and location above.'
```

Comment:
Suggest moving away from "here" and "above", because we do not necessarily know what order the fields will show up.

Here's one possibility: 
```suggestion
          'The data store id, when project id and location are provided as separate options. Alternatively, the full path to the data store should be provided in the form: "projects/{project}/locations/{location}/collections/default_collection/dataStores/{data_store}'.
```

Reply:
Done, thanks.